addons/jest/src/register.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import addons from '@storybook/addons';
 
-import PanelTitle from './components/PanelTitle';
+// import PanelTitle from './components/PanelTitle';
```

Comment:
The Title as component idea is kinda broken, so I simplified it to just text.

Will visit this problem again later, and provide a proper solution to the "which addons are currently relevant to display in the tabs?"